<!doctype html>
<html>
  <head>
    <title>LGSF Dashboard</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <link rel="shortcut icon" href="https://democracyclub.org.uk/static/images/logo_icon.png">
  </head>
  <body>
    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  


  <h2 id="2023-08-21-13-15">2023-08-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>5 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-08-21 13:15:03.077249</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-08-21 13:15:08.307816</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:15:03] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:15:04] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:15:05] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[13:15:08] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-08-14-14-14">2023-08-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-08-14 14:14:21.538173</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-08-14 14:14:24.238806</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:14:21] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:14:22] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:14:23] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:14:24] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-08-07-15-10">2023-08-07</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-08-07 15:10:40.470036</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-08-07 15:10:43.384024</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[15:10:40] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[15:10:41] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[15:10:42] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[15:10:43] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-07-31-14-56">2023-07-31</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-31 14:56:49.209499</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-31 14:56:52.495881</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:56:49] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:56:50] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:56:51] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:56:52] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-07-24-14-21">2023-07-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-24 14:21:07.515137</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-24 14:21:11.145267</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:21:07] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:21:08] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:21:09] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:21:10] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
[14:21:11] Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-07-17-14-06">2023-07-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-17 14:06:54.533042</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-17 14:06:58.853631</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:06:54] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
[14:06:56] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:06:57] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:06:58] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-07-10-13-25">2023-07-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-10 13:25:47.196150</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-10 13:25:49.912261</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:25:47] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[13:25:48] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[13:25:49] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-07-03-14-43">2023-07-03</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-03 14:43:36.039362</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-03 14:43:39.039219</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:43:36] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[14:43:37] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:43:38] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-06-26-15-28">2023-06-26</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-06-26 15:28:37.858654</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-06-26 15:28:41.087189</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[15:28:37] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
[15:28:38] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[15:28:39] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[15:28:40] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
[15:28:41] Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-06-19-14-09">2023-06-19</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-06-19 14:09:41.936777</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-06-19 14:09:44.761245</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:09:41] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
[14:09:42] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:09:43] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:09:44] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-06-12-13-42">2023-06-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-06-12 13:42:13.606798</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-06-12 13:42:16.535390</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:42:13] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:42:14] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:42:15] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[13:42:16] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-06-05-13-31">2023-06-05</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-06-05 13:31:14.987366</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-06-05 13:31:17.744420</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:31:14] Fetching Scraper for: SEG                              handlers.py:23
[13:31:15] Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
[13:31:16] Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[13:31:17] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-05-29-13-05">2023-05-29</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-05-29 13:05:54.090177</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-05-29 13:05:57.784361</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:05:54] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[13:05:55] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[13:05:56] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
[13:05:57] Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-05-22-14-18">2023-05-22</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-05-22 14:18:59.617945</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-05-22 14:19:02.999908</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:18:59] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:19:00] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:19:01] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:19:02] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-05-15-13-45">2023-05-15</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-05-15 13:45:43.656767</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-05-15 13:45:47.024108</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:45:43] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:45:44] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:45:45] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[13:45:46] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
[13:45:47] Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-05-08-13-59">2023-05-08</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-05-08 13:59:05.503013</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-05-08 13:59:08.221686</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:59:05] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:59:06] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:59:07] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[13:59:08] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-05-01-13-56">2023-05-01</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-05-01 13:56:30.201194</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-05-01 13:56:33.109166</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:56:30] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:56:31] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:56:32] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
           404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
[13:56:33] Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-04-24-13-42">2023-04-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-04-24 13:42:01.117540</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-04-24 13:42:04.055376</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 49, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 148, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 130, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:42:01] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[13:42:02] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:42:03] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
           404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
[13:42:04] Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-04-17-14-44">2023-04-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-04-17 14:44:59.714347</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-04-17 14:45:04.254983</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 46, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 134, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 123, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:44:59] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
[14:45:00] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:45:01] ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[14:45:04] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
           Finished attempting to scrape: SEG                        base.py:324
</pre>
  

  


  <h2 id="2023-04-10-15-00">2023-04-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-04-10 15:00:06.898158</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-04-10 15:00:11.140554</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 46, in run
    for councillor_html in self.get_councillors():
  File "/var/task/lgsf/councillors/scrapers.py", line 141, in get_councillors
    container = self.get_list_container()
  File "/var/task/lgsf/councillors/scrapers.py", line 134, in get_list_container
    self.base_url_soup = self.get_page(self.base_url)
  File "/var/task/lgsf/councillors/scrapers.py", line 123, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.somerset.gov.uk/we-have-moved/
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[15:00:06] Fetching Scraper for: SEG                              handlers.py:23
           Begin attempting to scrape: SEG                        handlers.py:27
[15:00:07] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[15:00:08] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from https://www.sedgemoor.gov.uk/councillors     base.py:42
[15:00:10] 404 Client Error: Not Found for url:                   handlers.py:36
           https://www.somerset.gov.uk/we-have-moved/                           
[15:00:11] Finished attempting to scrape: SEG                        base.py:324
</pre>
  


      </main>
      <footer class="ds-footer">
        <div class="ds-block-centered ds-text-centered ds-stack">
          <div class="ds-cluster-center">
            <ul>
              <li><a href="/lgsf-dashboard/api/failing.json">Failing JSON</a></li>
            </ul>
          </div>
          <div class="ds-copyright">
            <a href="https://democracyclub.org.uk/">
              <img src="https://DemocracyClub.github.io/design-system/images/logo_white_text.svg" alt="Democracy Club Home" />
            </a>
            <p>Copyright © 2021 Democracy Club</p>
            <p>Community Interest Company</p>
            <p>Company No: <a href="https://beta.companieshouse.gov.uk/company/09461226">09461226</a></p>
          </div>
        </div>
      </footer>
    </div>
  </body>
